capi/Makefile.am

```diff
@@ -1,4 +1,4 @@
-INCLUDES = -I$(top_srcdir) \
+AM_CPPFLAGS = -I$(top_srcdir) \
```

Comment:
You can remove AM_CPPFLAGS definition since  `-I$(top_srcdir)` should be automatic.  And `-I($top_srcdir)/interface` is unnecessary since you fixed the include locations.